### ISSUE TITLE
No picture shown on piwigo ng 1.0.2 

### ISSUE THREAD

**Issue by carnby77** [Piwigo/piwigo-flutter-app#108]
I have set up the piwigo server on my raspberry pi. It can create albums and upload photos from the website just fine.
But the mobile app piwigo ng version 1.0.2 has a problem. 
I can see any of the pictures on the album. i can upload to server . But the pictures on the mobile app appear as a broken sign.

Phone: Huawei Mate 10 Android 10.0.0
![WhatsApp Image 2022-04-14 at 12 22 51 AM](https://user-images.githubusercontent.com/89453256/163226257-2e193b78-9da7-4d5b-a5f3-b935c67edea7.jpeg)

**Comment by remi-martin** [comment]
Hi, the icon is here to show that there are no photos (thumbnail) in this album. Also, have you checked this issue #85 because it might correspond to your problem.

**Comment by zecamigo** [comment]
I have a very similar if not the same issue.

Context: 
Piwigo installed with Yunohost (LDAP)
Piwigo NG in Android, admin user logged in.

Album is created, photos are uploaded
However the photos are only visible by the webmaster, or on the website admin interface.

No matter what, I cannot make the photos visible.

This is different from #85 as the are no photos displayed at all in the app and website user interface. Count zero.
The folder is visible but with a broken image symbol as shown above.

Hope this helps. Available to provide more details and troubleshooting.

**Comment by zecamigo** [comment]
Hi thanks for replying!
 
> Than this issue concerns the [Piwigo repository](https://github.com/Piwigo/Piwigo/issues). It might be related to permissions, I'd recommend to check the photos and albums visibility options.

This happens only to photos uploaded through the Piwigo NG App on Android.
I've check permissions and visibility options they are all ok. Photos uploaded with the App are only visible to Admin and Webmaster in Admin view. I could not find a way to make them visible to normal users.

Photos uploaded through the Web interface are displayed correctly to all users with the right permissions, so it seems app specific.

**Comment by remi-martin** [comment]
Alright, in the newest updates, Piwigo API have recently added a lounge where the photos are stored after the upload. So we have to manually say to Piwigo server that we have finished uploading photos and that they can be added to the gallery. It seems it is not working well in your version of Piwigo NG.

Right now, I can't say anything more as I can't work on this right now. I'll give more information as soon as possible.

**Comment by zecamigo** [comment]
Thanks again for your time

> it is not working well in your version of Piwigo NG.

BTW Piwigo NG v1.0.3

> Right now, I can't say anything more as I can't work on this right now. I'll give more information as soon as possible.

No worries Rèmi, will be using the web interface as as a workaround in the meanwhile.

**Comment by zecamigo** [comment]
Fixed with Piwigo 12.3.0 as far as I can see.
https://github.com/Piwigo/Piwigo/releases/tag/12.3.0

**Comment by awardic** [comment]
I cant see a picture itself in the app, all thumbs are OK. When I'm logged in, pictures are visible. There is no problem in the web. Piwigo 12.3.0, php 7.4 (too much error on php 8.0), Apache 2.4.37. Upload in the web-version. URL pix.linth.net
![Screenshots](https://user-images.githubusercontent.com/34961427/185807622-ccb349bd-1760-428e-868c-98449f56503a.png)

**Comment by remi-martin** [comment]
So the fullscreen images are not working, I'll check that next week when I'll come back ^^.

**Comment by awardic** [comment]
Right, when I'm logged in, its working. There is no hurry for me but I'm probably not the only one with this error :(
